Rules/Zombie_Commands.as

```diff
@@ -2,6 +2,7 @@
 
 #include "Zombie_SoftBansCommon.as";
 #include "Zombie_GlobalMessagesCommon.as";
+#include "Zombie_Warns.as";

```

Comment:
Paired with my previous suggestion, this should be  ``#include "Zombie_WarnsCommon.as";`` instead